monitor/getBalances.js

```diff
@@ -91,24 +113,38 @@ async function main(bridgeMode) {
     const blockRewardAddress = await homeBridge.methods.blockRewardContract().call()
     const blockRewardContract = new web3Home.eth.Contract(BLOCK_REWARD_ABI, blockRewardAddress)
     logger.debug('calling blockReward.methods.mintedTotally')
-    const mintedCoins = await blockRewardContract.methods.mintedTotally().call()
+    const mintedCoins = await blockRewardContract.methods.mintedTotallyByBridge(COMMON_HOME_BRIDGE_ADDRESS).call()
     logger.debug('calling homeBridge.methods.totalBurntCoins')
     const burntCoins = await homeBridge.methods.totalBurntCoins().call()
 
     const mintedCoinsBN = new BN(mintedCoins)
     const burntCoinsBN = new BN(burntCoins)
     const totalSupplyBN = mintedCoinsBN.minus(burntCoinsBN)
     const foreignErc20BalanceBN = new BN(foreignErc20Balance)
+    const halfDuplexErc20BalanceBN = new BN(foreignHalfDuplexErc20Balance)
+
+    const diff = foreignErc20BalanceBN
+      .plus(halfDuplexErc20BalanceBN)
+      .minus(totalSupplyBN)
+      .toFixed()
+
+    let foreign = {
+      erc20Balance: Web3Utils.fromWei(foreignErc20Balance)
+    }
+
+    if (displayHalfDuplexToken) {
+      foreign = {
+        ...foreign,
+        halfDuplexErc20Balance: Web3Utils.fromWei(foreignHalfDuplexErc20Balance)
```

Comment:
how about an idea to display the half duplex token balance after the emergency shutdown but in another field like `halfDuplexErc20BalanceAfterES`? This value will not be taken into account for the `diff` calculation but will be reported in the monitor's artefacts.

Reply:
That sounds good to me, I added the changes https://github.com/poanetwork/tokenbridge/pull/245/commits/22b1bf162377f7033d40ada2ba4a7f2866a583a8